File: app/src/main/java/com/rachelleignacio/listbucket/db/DbFlowDatabase.java
package com.rachelleignacio.listbucket.db;

import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Created by rachelleignacio on 3/6/17.
 */
@Database(name = DbFlowDatabase.NAME,
        version = DbFlowDatabase.VERSION,
        foreignKeyConstraintsEnforced = true)
public class DbFlowDatabase {
    public static final String NAME = "ListBucketDatabase";
    public static final int VERSION = 1;
}

File: app/src/main/kotlin/com/rachelleignacio/listbucket/db/DbFlowDatabase.kt
package com.rachelleignacio.listbucket.db

import com.raizlabs.android.dbflow.annotation.Database

/**
 * Created by rachelleignacio on 8/4/17.
 */
@Database(name = DbFlowDatabase.NAME,
        version = DbFlowDatabase.VERSION,
        foreignKeyConstraintsEnforced = true)
object DbFlowDatabase {
    const val NAME = "ListBucketDatabase"
    const val VERSION = 1
}